### DISCUSSION TITLE
醫學影像數據集集錦

### DISCUSSION THREAD

**Discussion by ching020721** [1101-GS4719-C/Python-Community-GS4719-C#890] [優秀專案]
如果有同樣在做醫學影像處理的同學有需要進行實作練習就一定會很需要各種datasets，這個作者就提供了很多數據集供大家做使用，有提供數據集的論文與下載連結，簡介也很清楚明瞭，值得存起來日後有需要都可以當作參考使用。

https://github.com/linhandev/dataset

**Comment by Kotori0501** [comment]
感謝分享
這學期有和同學組了跨校專題，主題跟醫療影像挺有關聯的，希望可以好好使用此連結。

**Comment by ruthchen1020** [comment]
我覺得這個對一些人的專題十分有用，雖然我不需要但有這樣的資料庫之得可以幫助很多人!
